packages/camera/camera/lib/camera.dart

```diff
@@ -4,6 +4,7 @@
 
 export 'package:camera_platform_interface/camera_platform_interface.dart'
     show
+        AppleCaptureDeviceType,
```

Comment:
This is unlikely to be something we want to add at this layer. The goal of the app-facing package is to provide abstractions, not directly mirror every low-level detail of every platform, as the latter will result in an extremely confusing API.

We have patterns around having high-level abstractions, and then providing platform-specific subclasses with platform details if necessary, which would seem like a better option here. The first thing to evaluate is whether we actually need platform-specific details at all here; having different types of cameras isn't unique to iOS, so a design proposal that surveys the relevant APIs/distinctions across platforms would be the best place to start.